.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
+updates:
+  - package-ecosystem: cargo
+    directory: "/"
+    schedule:
+      interval: weekly
+      day: sunday
+      time: "7:00"
+    open-pull-requests-limit: 10
+    target-branch: master
+    labels: [auto-dependencies]
```

Comment:
I think we should add a new label for tickets about auto-dependency

Reply:
Created: https://github.com/apache/arrow-datafusion/labels/auto-dependencies